leku/src/main/res/layout/leku_activity_location_picker.xml

```diff
@@ -128,7 +130,12 @@
               tools:ignore="UnusedAttribute" />
 
           <include layout="@layout/leku_layout_coordinates" />
-          <include layout="@layout/leku_layout_address" />
+          <include layout="@layout/leku_layout_address"
```

Comment:
this prevents clipping on with the close button and maintains the legacy layout as it is